instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/semconv/rpc/RpcServerMetrics.java

```diff
@@ -72,10 +94,21 @@ public void onEnd(Context context, Attributes endAttributes, long endNanos) {
           context);
       return;
     }
+    Attributes attributes = state.startAttributes().toBuilder().putAll(endAttributes).build();
     serverDurationHistogram.record(
-        (endNanos - state.startTimeNanos()) / NANOS_PER_MS,
-        state.startAttributes().toBuilder().putAll(endAttributes).build(),
-        context);
+        (endNanos - state.startTimeNanos()) / NANOS_PER_MS, attributes, context);
+
+    Long rpcServerRequestBodySize =
+        RpcMessageBodySizeUtil.getRpcServerRequestBodySize(endAttributes, state.startAttributes());
+    if (rpcServerRequestBodySize != null && rpcServerRequestBodySize > 0) {
```

Comment:
`> 0` condition is weird. I think you can get rid of it by changing the type in getter from `int` -> `Long` so you can distinguish when getter does not implement this.

Reply:
Done.